minho/programmers/Level_1/6일차/체육복.js

```diff
@@ -0,0 +1,22 @@
+function solution(n, lost, reserve) {
+    let real_lost = lost.filter((el) => !reserve.includes(el)).sort((a, b) => a - b);
+    let real_reserve = reserve.filter((el) => !lost.includes(el)).sort((a, b) => a - b);
+    
+    let result = n - real_lost.length;
+    
+    real_lost.forEach((el) => {
+        
+         if(real_reserve.length === 0){
```

Comment:
```suggestion
         if (!real_reserve.length) {
```
결과값이 0이면은 false이기 때문에 !를 붙여주면 real_reserve.length가 0일때 조건문이 실행이 됩니다!